packages/stream-model-handler/src/model-handler.ts

```diff
@@ -73,13 +73,13 @@ export class ModelHandler implements StreamHandler<Model> {
    * @param state - Document state
    */
   async applyCommit(
+    api: StreamReaderWriter,
     commitData: CommitData,
-    context: Context,
     state?: StreamState
   ): Promise<StreamState> {
     if (state == null) {
       // apply genesis
-      return this._applyGenesis(commitData, context)
+      return this._applyGenesis(api, commitData)
```

Comment:
Why do you need to change order here? To make TS notice the change?

Reply:
More for uniformity. We can revert it, but given introduction of the new interfaces, seemed like a good time to change it.